src/api/entities/User.ts

```diff
@@ -32,14 +33,21 @@ export class User extends BaseModel {
   };
   id!: number;
   email!: string;
-  location: string | undefined;
-  phone: string | undefined;
+  firstName!: string;
+  lastName!: string;
+  location?: string;
+  phone?: string;
   role!: UserRole;
+  participantId!: number;
```

Comment:
participant Id is optional in the schema 
`participantId: z.number().optional()`

Reply:
Seems to need to be optional for the create account workflow. I won't dig into it for now.

Reply:
I think when a user is created, it always has a participantId, maybe we should make this column notNullable in schema